linetools/isgm/abssystem.py

```diff
@@ -108,6 +109,13 @@ def __init__(self, abs_type, radec, zabs, vlim, zem=0., NHI=0., sig_NHI=np.zeros
                 self.coord = SkyCoord(ra=radec[0], dec=radec[1])
         elif isinstance(radec,SkyCoord):
             self.coord = radec
+        elif isinstance(radec,basestring):
+            if ':' in radec:
+                self.coord = SkyCoord(radec, frame='fk5', unit=(u.hourangle, u.deg))
+            else:  # Add in :
```

Comment:
Can you test here that the string contains either '+' or '-'? Otherwise it will output a confusing error message for bad string input.